utils/android_utils.cake

```diff
@@ -0,0 +1,99 @@
+#load "./cake_utils.cake"
+#load "../build.cake"
+
+using Google.Apis.AndroidPublisher.v3;
```

Comment:
We're not using Google Apis - let's remove these statements and any .nuget reference to them.